src/firmware/linux/host/types/snp.rs

```diff
@@ -249,6 +249,71 @@ impl Default for SnpSetConfig {
     }
 }
 
+// Length defined in the Linux Kernel for the IOCTL.
+const VLEK_BUFFER_LEN: usize = 432;
+
+#[cfg(feature = "snp")]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[repr(C, packed)]
+/// Wrapped VLEK data.
+pub struct WrappedVlekHashstick<'a> {
+    /// Opaque data provided by AMD Key Distribution Server
+    /// (as described in SEV-SNP Firmware ABI 1.54, SNP_VLEK_LOAD)
+    pub data: &'a [u8], // 432 bytes of data
+}
```

Comment:
So the KDS provides bytes not an actual x509 cert?

Reply:
Correct. It is a wrapped hashstick